sale_order_volume/models/sale_order.py

```diff
@@ -9,17 +9,37 @@ def _compute_volume(order_line):
     return order_line.product_id.volume * order_line.product_uom_qty
 
 
+def _compute_pallet_count(volume, pallet_volume):
+    if not pallet_volume or not volume:
+        return 0
+    if volume <= pallet_volume:
+        return 1
+    if volume % pallet_volume == 0:
+        return volume // pallet_volume
+    return (volume // pallet_volume) + 1
```

Comment:
I lean towards `if .. elif .. else` structure rather than `if .. return` sequence but it's a matter of taste.